cli/cmd/scan.go

```diff
@@ -78,6 +79,8 @@ func init() {
 		"export vulnerability scan results to the backend")
 	scanCmd.Flags().Bool("cis-docker-benchmark-scan", false,
 		"enables CIS docker benchmark scan. (relevant only for image source type)")
+	scanCmd.Flags().Bool("ignore-no-fix", false, "ignore vulnerabilities that are not fixed")
```

Comment:
maybe: ignore vulnerabilities that have no fix available

Reply:
ack